docs/fundamentals/networking/http/httpclient-guidelines.md

```diff
@@ -49,8 +49,8 @@ To summarize recommended `HttpClient` use in terms of lifetime management, you s
 
 - In .NET Framework, use <xref:System.Net.Http.IHttpClientFactory> to manage your `HttpClient` instances. If you don't use the factory and instead create a new client instance for each request yourself, you can exhaust available ports.
 
-    > [!TIP]
-    > If your app requires cookies, consider disabling automatic cookie handling or avoiding <xref:System.Net.Http.IHttpClientFactory>. Pooling the <xref:System.Net.Http.HttpMessageHandler> instances results in sharing of <xref:System.Net.CookieContainer> objects. Unanticipated <xref:System.Net.CookieContainer> object sharing often results in incorrect code.
```

Comment:
I removed the `consider disabling automatic cookie handling` recommendation since we did not explain how to do it. I assume the note meant disabling `UseCookies` and managing `Cookie` and `Set-Cookie` headers manually, which is a quite advanced and inconvenient technique. I wouldn't recommend it without deeper elaboration.